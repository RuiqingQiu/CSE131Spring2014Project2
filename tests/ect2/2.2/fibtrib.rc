function : int main(){
   int i = 1, j = 1;
   cout << 0 << " " << 1 << " ";
   while (j > 0){
      cout << j << " ";
      auto k = j;
      j = j + i;
      i = k;
   }
   cout << endl;
   
   int k = j = 1;
   i = 0;
   cout << 0 << " " << 1 << " ";
   while (k > 0){
      cout << k << " ";
      auto l = k;
      k = k + i + j;
      i = j;
      j = l;
   }
   cout << endl;

   return 0;
}
