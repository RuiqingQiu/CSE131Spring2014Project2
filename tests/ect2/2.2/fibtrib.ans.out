0 1 1 2 3 5 8 13 21 34 55 89 144 233 377 610 987 1597 2584 4181 6765 10946 17711 28657 46368 75025 121393 196418 317811 514229 832040 1346269 2178309 3524578 5702887 9227465 14930352 24157817 39088169 63245986 102334155 165580141 267914296 433494437 701408733 1134903170 1836311903 
0 1 1 2 4 7 13 24 44 81 149 274 504 927 1705 3136 5768 10609 19513 35890 66012 121415 223317 410744 755476 1389537 2555757 4700770 8646064 15902591 29249425 53798080 98950096 181997601 334745777 615693474 1132436852 2082876103 
